CONTRIBUTING.md

```diff
@@ -262,6 +245,120 @@ or
+
+* Pick up a Linux or MacOS.
+* Install Python 3.7, 3.8 or 3.9 with [poetry](https://github.com/python-poetry/poetry) for package management
+* Fork the AWS SDK for pandas repository and clone that into your development environment
+
+* Then run the command bellow to install all dependencies:
+
+``poetry install``
+
+* Go to the ``test_infra`` directory
+
+``cd test_infra``
+
+* Install CDK dependencies:
+
+``poetry install``
+
+* [OPTIONAL] Set AWS_DEFAULT_REGION to define the region the Ray Test environment will deploy into. You may want to choose a region which you don't currently use:
+
+``export AWS_DEFAULT_REGION=ap-northeast-1``
+
+* Go to the ``scripts`` directory
+
+``cd scripts``
+
+* Deploy the `ray` CDK stack.
+
+``./deploy-stack.sh ray``
+
+* Configure Ray Cluster 
+
+``vi ray-cluster-config.yaml`` 
+
+```
+# Update the following file to match your enviroment
+# The following is an example
+cluster_name: ray-cluster
+
+initial_workers: 2
+min_workers: 2
+max_workers: 2
+
+provider:
+    type: aws
+    region: us-east-1 # change region as required
+    availability_zone: us-east-1a,us-east-1b,us-east-1c # change azs as required
+    security_group:
+        GroupName: ray_client_security_group
+    cache_stopped_nodes: False
+
+available_node_types:
+  ray.head.default:
+    node_config:
+      InstanceType: r5n.2xlarge # change instance type as required
+      IamInstanceProfile:
+        Arn: arn:aws:iam::{UPDATE YOUR ACCOUNT ID HERE}:instance-profile/ray-cluster-instance-profile
+      ImageId: ami-0ea510fcb67686b48 # latest ray images -> https://github.com/amzn/amazon-ray#amazon-ray-images 
+      NetworkInterfaces:
+        - AssociatePublicIpAddress: True
+          SubnetId: {replace with subnet within above AZs}
+          Groups: [{ID of group `ray_client_security_group` created by the step above}]
+          DeviceIndex: 0
+
+  ray.worker.default:
+      min_workers: 2
+      max_workers: 2
+      node_config:
+        InstanceType: r5n.2xlarge
+        IamInstanceProfile:
+          Arn: arn:aws:iam::{UPDATE YOUR ACCOUNT ID HERE}:instance-profile/ray-cluster-instance-profile
+        ImageId: ami-0ea510fcb67686b48 # latest ray images -> https://github.com/amzn/amazon-ray#amazon-ray-images 
+        NetworkInterfaces:
+          - AssociatePublicIpAddress: True
+            SubnetId: {replace with subnet within above AZs}
+            Groups: [{ID of group `ray_client_security_group` created by the step above}]
+            DeviceIndex: 0
+
+setup_commands:
+- pip install "awswrangler[distributed]==3.0.0a2"
+- pip install pytest
+
+```
+
+* Create Ray Cluster 
+``ray up -y ray-cluster-config.yaml``
+
+* Push Load Tests to Ray Cluster
+``ray rsync-up ray-cluster-config.yaml tests/load /home/ubuntu/``
+
+* Submit Pytest Run to Ray Cluster
+```
+echo '''
+import os
+
+import pytest
+
+args = "-v load/"
+
+os.environ["AWS_DEFAULT_REGION"] = "us-east-1" # Replace with your region
```

Comment:
Perhaps we should suggest something like:
```
os.getenv("AWS_DEFAULT_REGION", "us-east-1")
```
to avoid overriding the region if it has been set using an environment variable as suggested on line 269?

Reply:
Good call, i'm going to add a conditional check for that.